### PR TITLE
Build and package headers

### PR DESCRIPTION
**Pull request by berkus**
Package headers during CircleCI build

### CODE REVIEW

**Review by rusmonster — CHANGES_REQUESTED** [review]
```
➜  twilio-boost-build git:(fix/package-headers) circleci config validate
Error: Error calling workflow: 'build-and-deploy'
Error calling job: 'build-headers'
Error calling command: 'build'
Type error for argument platform: expected type: enum ("android" "linux" "linux-cxx11-abi-disabled" "ios" "osx"), actual value: "headers" (type string)
```

**Comment by berkus** [comment]
Updated.

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ aliases:
       platform:
         description: "Platform to build"
         type: enum
-        enum: ["android", "linux", "linux-cxx11-abi-disabled", "ios", "osx"]
+        enum: ["headers", android", "linux", "linux-cxx11-abi-disabled", "ios", "osx"]
```
Comment (Gray-Wind) [inline .circleci/config.yml]: You have missed quote in android.

**Review by rusmonster — CHANGES_REQUESTED** [review]
```
➜  twilio-boost-build git:(fix/package-headers) circleci config validate
Error: Error calling workflow: 'build-and-deploy'
Error calling job: 'build-android'
Error calling command: 'build'
Type error for argument platform: expected type: enum ("headers" "android\"" "linux" "linux-cxx11-abi-disabled" "ios" "osx"), actual value: "android" (type string)
```

File: .circleci/config.yml (hunk 1 above)
Comment (berkus) [reply .circleci/config.yml]: :)

**Review by rusmonster — APPROVED** [review]

**Pull request merged**